environment.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
 - joblib
 - memory_profiler
 - einops
+- zarr
```

Comment:
This file have been removed during the code sprint, you could update your branch.
You need to add zarr dependency in [setup.py](https://github.com/braindecode/braindecode/blob/264c77558d1a7ae337e92370bb231597807a2bf8/setup.py#L35).